taxcalc/calculate.py

```diff
@@ -58,6 +58,9 @@ def __init__(self, parameters, records, **kwargs):
         self._records = (records if not isinstance(records, str) else
                          Records.from_file(records, **kwargs))
 
+        while self._records.current_year < self._parameters.current_year:
```

Comment:
What I mean to say is that a user may unintentionally get incremented records when they accidentally specified the wrong data. In that case, it would be good to give an error message instead of just incrementing the records until the years match.